### PR TITLE
Disable MSAA for iOS and Android

### PR DESCRIPTION
**Pull request by CedricGuillemet**
Move bgfx reset flags into an header per platform.
MSAA is disabled for Android and iOS. Comments/links on why it's disabled are in the code.

- tested on Android, Windows
- tested on MacOS & iOS with highlighting

### CODE REVIEW

File: Core/Graphics/Source/GraphicsImpl.cpp [hunk 1]
```diff
@@ -30,7 +31,7 @@ namespace Babylon
 
         auto& init = m_state.Bgfx.InitState;
         init.type = s_bgfxRenderType;
-        init.resolution.reset = BGFX_RESET_VSYNC | BGFX_RESET_MSAA_X4 | BGFX_RESET_MAXANISOTROPY;
```
Comment (bghgary) [inline Core/Graphics/Source/GraphicsImpl.cpp:33]: I'm not sure we should split this into parts. The intention is that they are the same. Can you think of a reason why we want the reset flags to be different on purpose? IMO we can just use #ifdefs here and put a comment saying it's broken for Android/iOS like you did but in one place.

File: Core/Graphics/CMakeLists.txt [hunk 2]
```diff
@@ -26,15 +26,17 @@ set(SOURCES
     "Source/GraphicsImpl_${PLATFORM_NAME}.${BABYLON_NATIVE_PLATFORM_IMPL_EXT}"
     "Source/GraphicsImpl_${GRAPHICS_API}.cpp"
     "Source/SafeTimespanGuarantor.cpp"
-    "Source/SafeTimespanGuarantor.h")
+    "Source/SafeTimespanGuarantor.h"
+    "Source/${PLATFORM_NAME}/GraphicsFlags.h")
```
[the remark this answers is not in the document]
Comment (CedricGuillemet) [reply Core/Graphics/CMakeLists.txt:30]: GraphicsPlatform.h is public. It means bgfx.h must be included and public. I wanted to keep bgfx private here.

File: Core/Graphics/Source/GraphicsImpl.cpp (hunk 1 above)
Comment (CedricGuillemet) [reply Core/Graphics/Source/GraphicsImpl.cpp:33]: Using #ifdef was my first choice but you proposed to put the flags in header, per platform.

Comment (bghgary) [reply Core/Graphics/Source/GraphicsImpl.cpp:33]: Sorry, yes, I did, but I wasn't thinking about what the intention was.

Comment (bghgary) [reply Core/Graphics/Source/GraphicsImpl.cpp:33]: Let me take care of this since I broke the Android version.

**Pull request closed without merging**